ns-theme-check.php

```diff
@@ -28,6 +28,32 @@
 // Load admin.
 require_once NS_THEME_CHECK_DIR . '/inc/admin.php';
 
+$GLOBALS['ns_theme_check_standards'] = array(
+    'wordpress' => array(
+        'label'   => 'WordPress',
+        'default' => 0,
+        ),
+    'wordpress-core' => array(
+        'label'   => 'WordPress-Core',
+        'default' => 0,
+        ),
+    'wordpress-theme' => array(
+        'label'   => 'WordPress-Theme',
+        'default' => 1,
+        ),
+    'wordpress-docs' => array(
+        'label'   => 'WordPress-Docs',
+        'default' => 0,
+        ),
+    'wordpress-extra' => array(
+        'label'   => 'WordPress-Extra',
+        'default' => 0,
+        ),
+    'wordpress-vip' => array(
+        'label'   => 'WordPress-VIP',
+        'default' => 0,
```

Comment:
I would add a description to all of them. You can find more info here: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards#standards-subsets